lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -201,12 +201,19 @@ def excludes(offending_files, cop_name, parent)
         ((cfg['Exclude'] || []) + offending_files).uniq
       end
 
-      def output_exclude_path(output_buffer, file, parent)
-        file_path = Pathname.new(file)
+      def output_exclude_path(output_buffer, exclude_path, parent)
+        # exclude_path is either relative path, an absolute path, or a regexp.
+        file_path = Pathname.new(exclude_path)
         relative = file_path.relative_path_from(parent)
         output_buffer.puts "    - '#{relative}'"
       rescue ArgumentError
+        file = exclude_path
         output_buffer.puts "    - '#{file}'"
+      rescue TypeError => e
+        raise if e.message != 'no implicit conversion of Regexp into String'
```

Comment:
All tests still pass if I remove this line. If you can come up with a spec example that fails without this line, add it. Otherwise remove the line.